### PR TITLE
Add roadmap

### PR DESCRIPTION
**Pull request by justinsb**
This is a strawman proposal for a roadmap.

Please rip it apart / add things @chrislovecnm @kris-nova @geojaz @yissachar  et al :-)

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kops/2039)

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+## Must-have features
+
+* Support for k8s 1.6
+* RBAC enabled by default
+* etcd version selection (probably etc3 by default)
+
+## Other features we are working on in the 1.6 timeframe
+
+* Support for GCE
+* Support for ContainerOS
```
Comment (kris-nova) [inline ROADMAP.md]: Container Linux?

**Comment by chrislovecnm** [comment]
We had a few updates from the last office hours.  Where the updates included in this version?  I can review later, but I do not have time this moment.

File: ROADMAP.md (hunk 1 above)
Comment (aledbf) [inline ROADMAP.md]: @kris-nova https://cloud.google.com/container-optimized-os

**Comment by kris-nova** [comment]
Also check out how `dep` is handling our roadmap.. in case we are interested in borrowing any ideas here at `kops` https://github.com/golang/dep/wiki/Roadmap

File: ROADMAP.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# ROADMAP
+
+# 1.6
+
+## Must-have features
+
+* Support for k8s 1.6
+* RBAC enabled by default
```
Comment (justinsb) [inline ROADMAP.md]: Perhaps not enabled, but definitely easily enabled

File: ROADMAP.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+* etcd version selection (probably etc3 by default)
```
Comment (justinsb) [inline ROADMAP.md]: Typo: etcd3, maybe etcd3 for new clusters, probably don't auto-upgrade users though

**Pull request merged**